File: web3rpc/rpc-specs/api/code-samples/javascript/personal/deriveAccount.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const url = "url";
    const path = "path";
    const pin = true;

    sdk.personal.deriveAccount(url, path, {pin}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/api/code-samples/python/personal/deriveAccount.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

url = "url"
path = "path"
pin = True

sdk = OpenSDK(host)
personal_response = sdk.personal.derive_account(url, path, pin)

print(personal_response)
